frontend/src/assets/styles/theme.ts

```diff
@@ -64,4 +64,31 @@ const scrollBar = css`
     }
 `;
 
-export { THEME_DARK, scrollBar };
+const hintMessage = css`
+    &::after {
+        position: absolute;
+        bottom: -40px;
+        left: 50%;
+
+        padding: 0.5rem;
+        border-radius: 5px;
+
+        background: ${({ theme }) => theme.gray_300};
+        color: ${({ theme }) => theme.white};
+
+        font-size: ${({ theme }) => theme.font_xs};
+
+        opacity: 0;
+        transform: translate(-50%);
+        transition: opacity 0.2s;
+        white-space: nowrap;
+    }
+
+    &:hover {
+        &:after {
```

Comment:
after and before should contain 2 colon
"::after"

Reply:
done